.github/workflows/inspektor-gadget.yml

```diff
@@ -471,6 +471,11 @@ jobs:
       group: no-simultaneous-test-integration-aro
     steps:
     - uses: actions/checkout@v3
+    - name: Setup go
+      uses: actions/setup-go@v3
+      with:
+        go-version: 1.18.3
```

Comment:
Thanks for this. I should have done this in my change. At some point we should `env` for this so we are sure we use same version everywhere.

Reply:
Great suggestion. Don!